zipline/data/loader.py

```diff
@@ -47,7 +47,7 @@
 
 nyse_cal = get_calendar('NYSE')
```

Comment:
Any way to move these out of module scope, so we don't need to initialize them if we don't need em?